TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TeleOp/FSMs/MainTeleOp.java

```diff
@@ -15,7 +17,8 @@
 import org.firstinspires.ftc.teamcode.TeleOp.Mechanisms.Slides;
 import org.firstinspires.ftc.teamcode.TeleOp.Mechanisms.TransferController;
 
-@TeleOp(name = "TeleOp-1.2.1")
+@TeleOp(name = "TeleOp: 1.3.1")
```

Comment:
this should be 1.3.0
when the minor version is updated the patch version resets

Reply:
Got it